lizmap/modules/lizmap/classes/qgisForm.class.php

```diff
@@ -459,6 +459,25 @@ public function setFormDataFromFields($feature)
         return $form;
     }
 
+    /**
+     * Converts the datetime to the format specified in the qgis Project.
+     *
+     * @param string $value The datetime to convert
+     * @param object $edittype The format of the date
+     */
+    public function convertDateTimeToFormat($value, $edittype)
+    {
+        if (!property_exists($edittype, 'options') || !property_exists($edittype->options, 'field_format')) {
+            return $value;
+        }
+        $dateTab = explode('-', $value);
```

Comment:
You have the code handling this specific date format (yyyyMMdd) but I'd like it to be more generic and handle other formats like in this screenshot showing QGIS customizing date part.

![image](https://user-images.githubusercontent.com/2145040/92700469-2069ec00-f34f-11ea-8939-c504d6005f75.png)